src/go/rdctl/cmd/api.go

```diff
@@ -0,0 +1,199 @@
+/*
+Copyright © 2022 SUSE LLC
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+		http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package cmd
+
+import (
+	"bytes"
+	"fmt"
+	"io/ioutil"
+	"github.com/spf13/cobra"
+	"regexp"
+	"strings"
+	"text/template"
+)
+
+var apiSettings struct {
+	Method       string
+	InputFile    string
+	RawFields    []string
+	CookedFields    []string
+}
+
+// apiCmd represents the api command
+var apiCmd = &cobra.Command{
+	Use:   "api",
+	Short: "Run API endpoints directly",
+	Long: `Runs API endpoints directly.
+Default method is PUT if a body or input file is specified, GET otherwise.
+
+Two ways of specifying a body:
+1. --input FILE: Like the existing preferences file. Does not support interpolation.
+
+2. --raw-field|-f name=value : interpolates value as is into the body
+	 --field|-F name=value: "smartly" interpolates value into the body:
```

Comment:
This is misaligned:
```
2. --raw-field|-f name=value : interpolates value as is into the body
	 --field|-F name=value: "smartly" interpolates value into the body:
```
(Actually, it's _differently_ misaligned in my terminal and here in the review; best to just stick with pure spaces…)

Reply:
I've dropped those two fields because they make more sense for query parameters, and we don't have any of those yet.